egs/swbd/s5c/local/nnet3/tuning/run_tdnn_3c.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# 3c is as a, but uses xconfig.
```

Comment:
This naming scheme is not what we normally use.  It should be just linear: a,b,c, ..., [or 1a,1b,1c,...] then 2a,2b,2c... and so on.

The link points to run_tdnn_c.sh but I don't see that file..